src/middleware/sentry.js

```diff
@@ -0,0 +1,46 @@
+const Sentry = require('@sentry/node');
+
+const { systemInfo } = require('../logger');
+
+const replaceIds = (string) => {
+	if (string) {
+		return string.replace(/[a-f\d]{24}/ig, '[id]');
+	}
+	return string;
+};
+
+module.exports = (app) => {
+	const DSN = app.get('EDITOR_BACKEND_SENTRY_DSN');
+	if (DSN) {
+		systemInfo(`Sentry reporting enabled using DSN ${DSN}`);
+		Sentry.init({
+			dsn: DSN,
+			environment: app.get('NODE_ENV'),
+			release: app.get('version'),
+			integrations: [
+				new Sentry.Integrations.Console({
+					dsn: DSN,
```

Comment:
is the redefinition of the dsn here really required?

Reply:
is removed